src/stream.ts

```diff
@@ -44,7 +45,7 @@ export class WebRTCStream implements Stream {
   metadata: Record<string, any>;
   private readonly channel: RTCDataChannel;
 
-  source: Source<Uint8ArrayList> = pushable();
+  _src: Source<Uint8ArrayList> = pushable();
```

Comment:
Do I misremember "source" being part of the interface or has the interface changed?

Reply:
No, we just use a getter/setter pattern for accessing `source`.